ext/CFCoordinateReferenceSystemsProjExt/mappings.jl

```diff
@@ -0,0 +1,58 @@
+# Dictionary to map grid mapping names to their conversion functions
+const GRID_MAPPING_NAME_MAP = Dict{String,Function}(
+    "albers_conical_equal_area" => _albers_conical_equal_area,
+    "azimuthal_equidistant" => _azimuthal_equidistant,
+    "geostationary" => _geostationary,
+    "lambert_azimuthal_equal_area" => _lambert_azimuthal_equal_area,
+    "lambert_conformal_conic" => _lambert_conformal_conic,
+    "lambert_cylindrical_equal_area" => _lambert_cylindrical_equal_area,
+    "mercator" => _mercator,
+    "oblique_mercator" => _oblique_mercator,
+    "orthographic" => _orthographic,
+    "polar_stereographic" => _polar_stereographic,
+    "sinusoidal" => _sinusoidal,
+    "stereographic" => _stereographic,
+    "transverse_mercator" => _transverse_mercator,
+    "vertical_perspective" => _vertical_perspective,
+)
+
+const GEOGRAPHIC_GRID_MAPPING_NAME_MAP = Dict{String,Function}(
+    "rotated_latitude_longitude" => _rotated_latitude_longitude
+)
+
+# Copied from PROJ so we dont have to call it for something so trivial
+const PRIME_MERIDIAN_LONGITUDE = OrderedDict(
+    "Copenhagen" => 12.34399,
+    "Greenwich" => 0.0,
+    "Lisbon" => -9.0754862,
+    "Paris" => 2.5969213,
+    "Bogota" => -74.04513,
+    "Madrid" => -3.411455,
+    "Rome" => 12.27084,
+    "Bern" => 7.26225,
+    "Jakarta" => 106.482779,
+    "Ferro" => -17.4,
+    "Brussels" => 4.220471,
+    "Stockholm" => 18.03298,
+    "Athens" => 23.4258815,
+    "Oslo" => 10.43225,
+    "Paris RGS" => 2.201395,
+)
+
+const UNIT_DEGREE = "degree" 
+
+const DEFAULT_ELLIPSOIDAL_2D_AXIS_MAP = [
+    OrderedDict(
+        "name" => "Longitude",
+        "abbreviation" => "lon",
+        "direction" => "east",
+        "unit" => UNIT_DEGREE,
+    ),
+    OrderedDict(
```

Comment:
Let's make these LittleDicts (basically vectors) instead, apparently they are a lot faster

Reply:
Are they ordered? I just wanted to keep the output organised and non random

Reply:
LittleDicts are ordered yes

Reply:
Ok will switch

Reply:
I made a pr on top of this that changes everything to littledict

Reply:
Yeah let's merge it, I realised I forgot that yesterday